docs/guides/Redux-Integration.md

```diff
@@ -1,7 +1,13 @@
 # Redux Integration
 
-To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator. Your navigation prop must provide the current state, as well as access to a dispatcher to handle navigation options.
+### Roadmap for redux integration
```

Comment:
Instead of Roadmap I'd write "Overview"

Reply:
Just to keep consistent title case please rephrase to "Overview of Redux Integration"